message_definitions/v1.0/common.xml

```diff
@@ -3668,7 +3668,9 @@
       <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
       <field type="float" name="press_abs" units="hPa">Absolute pressure</field>
       <field type="float" name="press_diff" units="hPa">Differential pressure 1</field>
-      <field type="int16_t" name="temperature" units="cdegC">Temperature</field>
+      <field type="int16_t" name="temperature_press_abs" units="cdegC">Temperature</field>
```

Comment:
you can't rename an exsisting field (or change the type actually), it breaks all compatability

Reply:
fixed